index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Awesome book</title>
+</head>
+
+<body>
+    <h1>Awesome Books</h1>
+    <ul id="book-list"></ul>
+
+    <div class="text_input">
+        <div class="container">
+            <form class="Book">
+                <div>
+                    <label for="Title">Title</label>
+                    <input type="text" name="book" id="Title" class="title" />
+                </div>
+                <div>
+                    <label for="Author">Author</label>
+                    <input type="text" name="auth" id="Author" class="author" />
+                </div>
```

Comment:
- It would be great to have limit of characters because i tried to send inputs in blank and it saves into the list. Kindly fix this.